Cython/Debugger/Tests/test_libcython_in_gdb.py

```diff
@@ -265,9 +265,9 @@ def test_locals(self):
         self.break_and_run('int(10)')
 
         result = gdb.execute('cy locals', to_string=True)
-        assert 'a = 0', repr(result)
-        assert 'b = (int) 1', result
-        assert 'c = (int) 2' in result, repr(result)
+        self.assertIn('a = 0', result)
+        self.assertIn('b = (int) 1', result)
+        self.assertIn('c = (int) 2', result)
```

Comment:
I think that [this commit](https://github.com/cython/cython/commit/3aec761b21b13bf29a98575a0e32a665ba934986#diff-96da2032b81feec75e6282d512c63bf16d331b3fa4865d97317a27b3d31a0d1dL218-R220) was not reviewed and, accordingly, introduced the wrong behavior when it is checked not for the presence of a string in the result, but whether the string will return True as a boolean value.

Reply:
Yeah, seems reasonable.